index.html

```diff
@@ -3,8 +3,29 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
     <title>Document</title>
+    <link rel="styles" href="css/styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <section>
+      <h2>What is Riesling?</h2>
+      <button>Show Answer</button>
+      <p>A Grape variety, that was esay</p>
+    </section>
+    <section>
+      <h3></h3>
```

Comment:
maybe stay on h2, because the cards in the sections will be the same structure as the card before.